integration-tests/src/test/java/com/redhat/parodos/integration/notification/common/NotificationTestBuilder.java

```diff
@@ -0,0 +1,78 @@
+package com.redhat.parodos.integration.notification.common;
+
+import com.redhat.parodos.notification.sdk.api.ApiClient;
+import com.redhat.parodos.notification.sdk.api.ApiException;
+import com.redhat.parodos.notification.sdk.api.NotificationRecordApi;
+import com.redhat.parodos.notification.sdk.model.NotificationRecordResponseDTO;
+import com.redhat.parodos.notification.sdk.model.PageNotificationRecordResponseDTO;
+import com.redhat.parodos.sdkutils.NotificationServiceSdkUtils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class NotificationTestBuilder {
```

Comment:
There are duplicates between `NotificationTestBuilder` to `WorkFlowTestBuilder`
Consider to move those duplicates to common parent static class
(`public class NotificationTestBuilder extends BaseTestBuilder`)